magellan-library/src/main/java/com/wealthfront/magellan/view/ActionBarModifier.java

```diff
@@ -0,0 +1,38 @@
+package com.wealthfront.magellan.view;
+
+import android.view.Menu;
+import org.jetbrains.annotations.NotNull;
+import androidx.annotation.ColorRes;
+
+public interface ActionBarModifier {
```

Comment:
This needs to be in Java since this is a public interface that consumers of the library would need to implement. If we use @JvmDefault here instead, consumers would need to enable `-Xjvm-default` option or else would run into a compile time error. So instead i use a java interface here to maintain compatibility.

```
Inheritance from an interface with '@JvmDefault' members is only allowed with -Xjvm-default option
```